public/javascripts/app/collections/SnapshotIdModels.js

```diff
@@ -16,7 +14,7 @@ SnapshotIdModelsMod.factory('SnapshotIdModels', [
         class SnapshotIds extends BaseCollection {
             constructor(models){
                 super();
-                this.models = models.map((snapshot) => new SnapshotIdModel.getModel(snapshot)).sort(this.comparator);
+                this.models = models.map((snapshot) => SnapshotIdModel.getModel(snapshot)).sort(this.comparator);
```

Comment:
`getModel` calls `new` inside so you get an error. Presumably whatever ancient version of Babel or something we had was totally fine with this before